ide/provisioning/install-darwin-packages.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/zsh
```

Comment:
can you make it either bash or sh? zsh is great, but not natively installed in many places.

Reply:
Or is it on osx?

Reply:
Changed it to bash. Indeed, in macOS 12 zsh was not preinstalled (or default shell) yet.